spec/models/rules_of_origin/scheme_spec.rb

```diff
@@ -221,6 +221,10 @@
       it { is_expected.to eql [] }
     end
 
+    context 'with show_proofs_for_geographical_areas' do
```

Comment:
This all feels unnecessary. Can't we just extend the list of attributes on the response_data that already exists? There's nothing behavioural, here?

Reply:
yes, we could expand the spec for the `applies_to_geographical_area?(area)`